src/core/CcpnSimulatorPlugin.js

```diff
@@ -0,0 +1,18 @@
+class CcpnSimulatorPlugin {
```

Comment:
Mhm, somehow this naming scheme seems weird, maybe `SimulatorPluginCCPN`

Reply:
Yes sounds better. We should also rename other occurences of camel-cased acronyms to uppercase I think.

Reply:
Yes, thats a great idea. Could be more difficult than this, because there are multiple referenzes in multiple projects, how we handle this? We need to create issues for this.